.github/workflows/ci-build.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
   tests:
     name: Unit Tests
     runs-on: ubuntu-latest
-    timeout-minutes: 8
```

Comment:
Sync with the master. Master has unittest improvement

Reply:
Hi @sarabala1979, my PR is already based on the latest master: 
<img width="422" alt="image" src="https://user-images.githubusercontent.com/6445305/181293376-73e7959d-dc2e-4e09-be4a-2febcd923336.png">

Reply:
apologies, there was a new commit as of 4 hours ago I did not have, but I don't think this should affect the unittests. Rebased 👍 
https://github.com/mcgrawia/argo-workflows/commit/b44981f174acebac239e4ecbdda845aa4338f5af